src/bin/vip-import-sql.js

```diff
@@ -135,7 +138,9 @@ command( {
 
 			await trackEventWithEnv( 'import_sql_command_queued' );
 
-			console.log( '🚧 🚧 🚧 Your sql file import is queued 🚧 🚧 🚧' );
+			console.log( '\n🚧 🚧 🚧 Your sql file import is queued 🚧 🚧 🚧' );
+			console.log( '--> Check the status of your import via the \`import_progress\` site meta in the VIP Go Admin Console!\n' );
```

Comment:
```suggestion
			console.log( '--> Check the status of your import via the \`import_progress\` site meta in the VIP internal console!\n' );
```

Avoiding using internal names outside. Also, we should remove this once we deploy a public version (outside of `-dev`)

Reply:
Oops, got it! I added a comment as well to remind ourselves to remove it later on.